meta-mender-core/recipes-bsp/grub-mender-grubenv/files/01_efi_secure_boot_grub.cfg

```diff
@@ -0,0 +1,16 @@
+get_efivar -f uint8 -s secured SecureBoot
+if [ "${secured}" = "1" ]; then
+    if [ -s "${prefix}/password.inc" ]; then
+        source "${prefix}/password.inc"
+    fi
+fi
+
+get_efivar -f uint8 -s unprovisioned SetupMode
+
+if [ "${unprovisioned}" = "1" ]; then
+    chainloader "${prefix}/LockDown.efi"
```

Comment:
Could you give a bit more background about the `password.inc` and `LockDown.efi` files? Could really use some comments here.

Reply:
Fixed in the latest push